### DISCUSSION TITLE
How do tags work?

### DISCUSSION THREAD

**Discussion by Budlyte** [advplyr/audiobookshelf#55] [Q&A] [answered]
I'm trying to add a tag to some books but can't get it to take.

**Comment by advplyr** [comment]
Have you tried again with the new update? I may have fixed a bug with that dropdown.
It functions just like Genre.

btw, if you don't mind taking a few screenshots of your collection, I would much prefer to use your bookshelf for the screenshots in the readme and elsewhere. The cover art for your books really makes the new series page look awesome.

**Comment by Budlyte** [comment]
I tried Genre to test. It works, but looks like it has a pre-defined list of selections.  If something doesn't already exist, I see no way to create it. Same for Tags, which doesn't have any pre-existing items.

For similar systems pressing Space or Tab would create the intended new item, or an area in the Settings exists where these lists are able to be manipulated.

**Comment by advplyr** [comment, marked as answer]
Tags can have spaces in them, so you need to press `Enter` to create the new tag. Tags don't have any pre-defined.

![addingtags](https://user-images.githubusercontent.com/67830747/134828157-78c984df-fb05-41ce-ba8b-9e368a0e5250.gif)

**Reply by Budlyte** [reply]
LOL  now I feel dumb.  Again
